.storybook/addons/codeEditorAddon/editor.js

```diff
@@ -29,30 +29,29 @@ export class EditorElement extends LitElement {
         height: 100%;
         display: flex;
         flex-direction: column;
-        background-color: rgb(37, 37, 38);
+        background-color: rgb(243, 243, 243);
       }
 
-      .tab-root {
-        height: 38px;
-      }
+      
 
       .editor-root {
         flex-basis: 100%;
       }
 
       .tab {
-        background-color: rgb(45, 45, 45);
-        color: rgba(255, 255, 255, 0.25);
-        font-family: 'Segoe UI', 'Segoe UI Web (West European)', 'Segoe UI', -apple-system, BlinkMacSystemFont, Roboto,
-          'Helvetica Neue', sans-serif;
-        font-size: 12px;
-        padding: 14px 18px;
+        background-color: rgb(236,236,236);
+        color: #616161;
+        font-family: -apple-system,BlinkMacSystemFont,sans-serif;
```

Comment:
Should import the shared-styles file and use the default-font-family variable

Reply:
Not sure I agree - the tab header should be a different font that matches vscode, rather than our default font

Reply:
What was the original font that needed to be overwritten?

Reply:
Not sure, this is the font-family used by vscode